Packs/FeedMitreAttackv2/Integrations/FeedMitreAttackv2/FeedMitreAttackv2.py

```diff
@@ -0,0 +1,370 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+from typing import List, Dict, Set, Optional
+import json
+import requests
+from stix2 import TAXIICollectionSource, Filter
+from taxii2client.v20 import Server, Collection, ApiRoot
+
+''' CONSTANT VARIABLES '''
+
+MITRE_TYPE_TO_DEMISTO_TYPE = {
+    "attack-pattern": "Attack Pattern",
+    "course-of-action": "Course of Action",
+    "intrusion-set": "Intrusion Set",
+    "malware": "Malware",
+    "tool": "Tool",
```

Comment:
we should consider moving this to constants in commonServer.
@bakatzir WDYT?

Reply:
moved - https://github.com/demisto/content/pull/12764